tailwind/Link.tsx

```diff
@@ -0,0 +1,190 @@
+export const BaseLink = forwardRef<HTMLAnchorElement, LinkProps>(function Link(
+  {
+    href,
+    children,
+    className,
+    hideArrow,
+    isPartiallyActive = true,
+    activeClassName = "text-primary",
+    customEventOptions,
+    ...props
+  }: LinkProps,
+  ref
+) {
+  const { asPath } = useRouter()
+  const { flipForRtl } = useRtlFlip()
+
+  if (!href) {
+    console.warn("Link component is missing href prop")
+    return <a {...props} />
+  }
+
+  const isActive = url.isHrefActive(href, asPath, isPartiallyActive)
+  const isDiscordInvite = url.isDiscordInvite(href)
+  const isPdf = url.isPdf(href)
+  const isExternal = url.isExternal(href)
+  const isInternalPdf = isPdf && !isExternal
+  const isHash = url.isHash(href)
+
+  // Get proper download link for internally hosted PDF's & static files (ex: whitepaper)
+  // Opens in separate window.
+  if (isInternalPdf) {
+    href = getRelativePath(asPath, href)
+  }
+
+  if (isDiscordInvite) {
+    href = new URL(DISCORD_PATH, SITE_URL).href
+  }
+
+  const commonProps = {
+    ref,
+    ...props,
+    className: cn(className, { [activeClassName]: isActive }),
+    href,
+  }
+
+  if (isExternal) {
+    return (
+      <a
+        target="_blank"
+        rel="noopener"
+        onClick={() =>
+          trackCustomEvent(
+            customEventOptions ?? {
+              eventCategory: `Link`,
+              eventAction: `Clicked`,
+              eventName: "Clicked on external link",
+              eventValue: href,
+            }
+          )
+        }
+        {...commonProps}
+      >
+        {children}
+        <VisuallyHidden>(opens in a new tab)</VisuallyHidden>
+        {!hideArrow && (
+          <RxExternalLink
+            className={cn("-me-1 inline h-6 w-6 p-1 align-middle", {
+              transform: flipForRtl,
+            })}
+          />
+        )}
+      </a>
+    )
+  }
+
+  if (isInternalPdf) {
+    return (
+      <NextLink
+        target="_blank"
+        rel="noopener"
+        // disable locale prefixing for internal PDFs
+        // TODO: add i18n support using a rehype plugin (similar as we do for
+        // images)
+        locale={false}
+        onClick={() =>
+          trackCustomEvent(
+            customEventOptions ?? {
+              eventCategory: `Link`,
+              eventAction: `Clicked`,
+              eventName: "Clicked on internal PDF",
+              eventValue: href,
+            }
+          )
+        }
+        {...commonProps}
+      >
+        {children}
+      </NextLink>
+    )
+  }
+
+  if (isHash) {
+    return (
+      <a
+        onClick={(e) => {
+          e.stopPropagation()
+          trackCustomEvent(
+            customEventOptions ?? {
+              eventCategory: "Link",
+              eventAction: "Clicked",
+              eventName: "Clicked on hash link",
+              eventValue: href,
+            }
+          )
+        }}
+        {...commonProps}
+      >
+        {children}
+      </a>
+    )
+  }
```

Comment:
<img width="1532" alt="image" src="https://github.com/user-attachments/assets/cf6986b5-172c-48f7-84bf-6326f226deba">

@pettinarip We were not using NextLink before, just Chakra for styling. A simple anchor should suffice here similar to `isExternal`, and this fixes the issue with the SkipNav button not actually moving the focus to the desired `#main-content` element.

Reply:
Ahh got it, thanks for the fix!